src/chunkserver/chunkserver.cpp

```diff
@@ -104,7 +109,6 @@ int ChunkServer::Run(int argc, char** argv) {
 
     // 打印参数
     conf.PrintConfig();
-    conf.ExposeMetric("chunkserver_config");
```

Comment:
why remove this?

Reply:
It has exposed at https://github.com/opencurve/curve/blob/6f5c85b1660aeb55d5c08e3d1eb354949ab7567a/src/chunkserver/chunkserver.cpp#L317

The expose here was added later, and the names of the two exposes are different, so the monitoring cannot be obtained